### PR TITLE
Initial release candidate

### PR DESCRIPTION
**Pull request by Jammjammjamm**

### CODE REVIEW

File: lib/onc_certification_g10_test_kit.rb [hunk 1]
```diff
@@ -95,16 +95,24 @@ class G10CertificationSuite < Inferno::TestSuite
       * SMART Launch URI: `#{SMARTAppLaunch::AppLaunchTest.config.options[:launch_uri]}`
       * OAuth Redirect URI: `#{SMARTAppLaunch::AppRedirectTest.config.options[:redirect_uri]}`
 
-      Systems must pass all tests in order to qualify for ONC certification.
+      For the multi-patient API, register Inferno with the following JWK Set
+      Url, if the server supports it:
```
Comment (arscan) [inline lib/onc_certification_g10_test_kit.rb:99]:
```suggestion
      For the multi-patient API, register Inferno with the following JWK Set
      Url:
```

File: lib/onc_certification_g10_test_kit.rb [hunk 2]
```diff
@@ -95,16 +95,24 @@ class G10CertificationSuite < Inferno::TestSuite
...
 
-      input_instructions %(
+    input_instructions %(
         Register Inferno as a SMART app using the following information:
 
         * Launch URI: `#{SMARTAppLaunch::AppLaunchTest.config.options[:launch_uri]}`
         * Redirect URI: `#{SMARTAppLaunch::AppRedirectTest.config.options[:redirect_uri]}`
 
+        For the multi-patient API, register Inferno with the following JWK Set
+        Url, if the server supports it:
```
Comment (arscan) [inline lib/onc_certification_g10_test_kit.rb:113]:
```suggestion
        For the multi-patient API, register Inferno with the following JWK Set
        Url:
```

**Review by arscan — APPROVED** [review]

File: lib/onc_certification_g10_test_kit/multi_patient_api.rb [hunk 3]
```diff
@@ -9,9 +9,14 @@ class MultiPatientAPIGroup < Inferno::TestGroup
 
     input_instructions %(
       Register Inferno as a bulk data client with the following information, and
-      enter the client id and client registration in the appropriate fields. This
-      set of tests only checks the Group export. Enter the group export
+      enter the client id and client registration in the appropriate fields.
+      This set of tests only checks the Group export. Enter the group export
       information in the appropriate box.
+
+      If the server supports it, register Inferno with the following JWK Set
```
Comment (arscan) [inline lib/onc_certification_g10_test_kit/multi_patient_api.rb:16]:
```suggestion
      Register Inferno with the following JWK Set
```

Comment (Jammjammjamm) [reply lib/onc_certification_g10_test_kit/multi_patient_api.rb]: I did this in a separate commit to deal with line wrapping.

**Pull request merged**